scripts/init.sh

```diff
@@ -157,6 +162,7 @@ start-frequency-interval)
     -lruntime=debug \
     --sealing=interval \
     --sealing-interval=${interval} \
+    --sealing-create-empty-blocks \   # Fork aware pool is only supported with this feature
```

Comment:
At least on my mac, the comment caused the command to fail. Perhaps move the comment to the top of command with a `# Fork aware pool is only supported with sealing-create-empty-blocks` more generically

```suggestion
    --sealing-create-empty-blocks \
```